style.css

```diff
@@ -147,6 +152,14 @@ ul.work {
   height: 10px;
 }
 
+.popup-img > img {
+  width: 100%;
```

Comment:
- [ ] The image is of lower quality with the full width and height, if you tried to shrink it a little bit as in the given design, it's going to look better 😉 
- [ ] Also, please fix the modal's styling on the mobile screen to match the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
### **Your design**
![Screenshot](https://user-images.githubusercontent.com/47625407/123022640-5ce67000-d3d6-11eb-9421-5a3dbaae11b8.png)